stable/mailhog/templates/deployment.yaml

```diff
@@ -44,6 +44,10 @@ spec:
             - name: smtp
               containerPort: 1025
               protocol: TCP
+          {{- if .Values.auth.enabled }}
+          args:
```

Comment:
do we still need this https://github.com/helm/charts/blob/582fb9e1eed44e26bceef3b5afac800f55ab65fd/stable/mailhog/templates/deployment.yaml#L33 ?

Reply:
probably not.

But, getting a look again, I think that also "existingSecret" value could be removed. 
If you populate it, the secret is not created at all. Then the added "arg" cannot work (because there isn't a file).
At the same time, also considering that my PR is wrong and the "arg" command has to be removed, I cannot understand (reading this chart) how can work the authentication system...

I do not know if I was clear and if could be another issue

Reply:
i'm testing here

Reply:
@g0blin79 you don't need to have the env variable it works without that

but the existingSecret is in the case you already created that. then you need to use that one instead of the hardcoded one